Week1/homework/js-exercises/logHello1.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var hwTurkish = "Merhaba, Dunya!";
+var hwDutch = "Hallo, Wereld!";
+var hwEnglish = "Hello, World!";
+var hwIndonesian = "Halo, Dunia!";
+var hwItalian =  "Ciao, Mondo!";
+var hwSpanish = "Hola, Mundo!";
+var hwGerman = "Hallo, Welt!";
+var hwFrench = "Bonjour, le Monde!";
+var hwChech = "Ahoj, Světe!";
+var hwRussian = "Привет, мир!";
+
+var hwList = [hwTurkish, hwSpanish, hwRussian, hwItalian, 
+    hwIndonesian, hwGerman, hwFrench, hwEnglish, hwDutch, hwChech]
+
+for (let index = 0; index < hwList.length; index++) {
+    console.log(hwList[index]);
+}
+
```

Comment:
nice that you used for loop, can just declare strings inside the array as well getting rid of the variables